packages/slate-react/src/components/editable.tsx

```diff
@@ -624,19 +624,52 @@ export const Editable = (props: EditableProps) => {
               ) {
                 const node = ReactEditor.toSlateNode(editor, event.target)
                 const path = ReactEditor.findPath(editor, node)
-                const start = Editor.start(editor, path)
-                const end = Editor.end(editor, path)
+                // Check if user clicks thrice
+                if (event.detail === 3) {
+                  // Check the closest element block above the selection
+                  const elem = Editor.above(editor, {
```

Comment:
Might be clearer if it was `const block` rather than `const elem`, since it's not returning a match if it's not a block?